go.mod

```diff
@@ -102,6 +102,7 @@ require (
 	github.com/Azure/go-autorest/autorest v0.11.28
 	github.com/googleapis/gax-go/v2 v2.7.0
 	github.com/grafana/gomemcache v0.0.0-20230316202710-a081dae0aba9
+	github.com/grafana/regexp v0.0.0-20221123153739-15dc172cd2db
```

Comment:
A few of them not yet
* https://go-review.googlesource.com/c/go/+/353711
* https://go-review.googlesource.com/c/go/+/354909
* https://go-review.googlesource.com/c/go/+/358756
* https://go-review.googlesource.com/c/go/+/377294

Removing since I don't think they're relevant for this PR.